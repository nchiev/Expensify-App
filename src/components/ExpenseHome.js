import React from 'react';
import ExpenseList from './ExpenseList'
import ExpenseSummary from './ExpenseSummary';
import ExpenseListFilter from './ExpenseListFilter';


const ExpenseHome = () => (
    <div>
        <ExpenseListFilter />
        <ExpenseSummary />
        <ExpenseList />
    </div>
);

export default ExpenseHome;