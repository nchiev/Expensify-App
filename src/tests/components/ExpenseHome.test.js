import React from 'react';
import { shallow } from 'enzyme';

import ExpenseHome from '../../components/ExpenseHome';

test('should render home component', () => {
    const wrapper = shallow(<ExpenseHome />);

    expect(wrapper).toMatchSnapshot();
});