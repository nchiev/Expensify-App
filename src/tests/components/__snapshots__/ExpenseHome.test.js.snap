// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home component 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Component />
  <Connect(ExpenseList) />
</div>
`;
